notebooks/book2/11/mcmc_traceplots_unigauss.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    print(f"Number of divergences (bad prior) = {info.is_divergent[500:,:].sum()}")
</pre><p>What is 500 here? Burn-in? if so, should we create a variable for it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I have created <code>burn_in</code> as global variable now </p>